src/query/plan/operator.hpp

```diff
@@ -464,6 +473,14 @@ class CreateExpand : public memgraph::query::plan::LogicalOperator {
   /// if the given node atom refers to an existing node (either matched or created)
   bool existing_node_;
 
+  std::string ToString() const override {
+    return "CreateExpand (" + input_symbol_.name() + ")" +
+           (edge_info_.direction == query::EdgeAtom::Direction::IN ? "<-" : "-") + "[" + edge_info_.symbol.name() +
+           ":" + dba_->EdgeTypeToName(edge_info_.edge_type) + "]" +
+           (edge_info_.direction == query::EdgeAtom::Direction::OUT ? "->" : "-") + "(" + node_info_.symbol.name() +
+           ")";
```

Comment:
use `fmt::format`

Reply:
same in other operators please

Reply:
which ones have you changed, I'm not sure if all need to be changed or?

Reply:
I’ve only changed the operators whose arguments are shown by `EXPLAIN` clauses (with this PR, `PROFILE` clauses also print arguments, and if both use `ToString` there’s no code duplication).

Reply:
Switched to `fmt::format` everywhere